docs/concepts/EvictionAndDisruptionBudget/README.md

```diff
@@ -0,0 +1,65 @@
+# Eviction & Placement Disruption Budget
+
+This document explains the concept of `Eviction` and `Placement Disruption Budget` in the context of the fleet.
+
+## Overview
+
+`Eviction` pertains to the act of removing resources from a target cluster propagated by a resource placement object from the hub cluster.
```

Comment:
Hi Arvind! Sorry for being nitpicky, but this part could use some paraphrasing; for example, there are many ways of removing resources from a selected cluster, and `CRPEviction` is actually an API we provided for removing resources (i.e., it is one way, and the recommended way, of performing the removal).